### PR TITLE
More stealthboys + new flags

### PR DESCRIPTION
**Pull request by OliOliOnsiPree**
## About The Pull Request
This PR adds in 2 new stealth boys, which also 'balances' the first one, which has now a 275 charge instead of a 300. Also with the new stealth boys adds a makeshift one that starts with a 175 charge, which is barely visble but still can been seen, mk2 stealth boy is longer timing, almost 1.5 more then the mk1. The makeshift can be made if someone wants to rp it out. Also this also adds a doctors bag into the sprites of drugs and new flags which include; ncrflag with snow, old legion flga with snow, old oasis flga with snow, and bos flag red with snow and follwoers flag with snow aswell, and the enclave flag too.
Basically this is more of adding sprites from https://github.com/iWuna/Fallout-13
Adds these in:
image
image
image

## Pre-Merge Checklist
- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

## Changelog
:cl:
add: new sprites, and two new stealth boys.
/:cl:

### CODE REVIEW

**Comment by BadAtThisGame302** [comment]
Follower flag.. remove this... NOW!!! 😭 😭

**Comment by OliOliOnsiPree** [comment]
> Follower flag.. remove this... NOW!!! 😭 😭

Most likely will never be used, the snow ones sadly.
